openstack/cce/v3/clusters/results.go

```diff
@@ -107,9 +107,15 @@ type CidrSpec struct {
 
 type EniNetworkSpec struct {
 	//Eni network subnet id
-	SubnetId string `json:"eniSubnetId" required:"true"`
```

Comment:
it's better to add more comments as `SubnetId` and `Cidr` will be deprecated in the feature.

Reply:
sure